src/products/components/ProductOrganization/ProductOrganization.tsx

```diff
@@ -156,6 +159,47 @@ export const ProductOrganization: React.FC<ProductOrganizationProps> = props =>
               id: "ccXLVi",
               defaultMessage: "Category",
             })}
+            {...(!categoryInputActive &&
+              !disabled && {
+                width: "100%",
+                __opacity: 0,
+                position: "absolute",
+              })}
+            onFocus={() => {
+              setCategoryInputActive(true);
+            }}
+            onBlur={() => {
+              setCategoryInputActive(false);
+            }}
+            startAdornment={val => {
+              if (categoryInputActive || disabled) {
+                return undefined;
+              }
+
+              let availableCategories = categories;
+
+              if (selectedProductCategory) {
+                availableCategories = [...categories, selectedProductCategory];
+              }
```

Comment:
suggestion: I think we should avoid using `let` if it can be solved with a ternary operator

```suggestion
              const availableCategories = selectedProductCategory ? [...categories, selectedProductCategory] : categories;
```

Reply:
you're right, thanks 😸